src/app/features/genes/components/gene-comparison-tool/gene-comparison-tool.component.ts

```diff
@@ -912,15 +912,20 @@ export class GeneComparisonToolComponent implements OnInit, AVI, OnDestroy {
     const tissue = gene.tissues.find((t) => t.name === tissueName);
     let size = 0;
     let color = '#F0F0F0';
+    const CIRCLE_MAX_SIZE = 50;
 
     if (tissue?.adj_p_val) {
       size = this.getCircleSize(tissue.adj_p_val);
     }
 
+    if (size === 0) {
+      size = CIRCLE_MAX_SIZE;
+    }
```

Comment:
What do you think about handling this case in `getCircleSize` instead, so all of the sizing is determined in the same function?